### PR TITLE
added input and button

### PR DESCRIPTION
**Pull request by benazuz**
BLA BLA

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
+    <input type="text"><button>OK</button>
```
Comment (eranshmil) [inline dist/index.html]: Please change to `Cancel`.

Comment (benazuz) [reply dist/index.html]: ok

File: dist/index.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
```
Comment (eranshmil) [inline dist/index.html]: Add class `black`.

File: dist/index.html (hunk 1 above)
Comment (benazuz) [inline dist/index.html]: great code

File: dist/index.html [hunk 3]
```diff
@@ -8,6 +8,7 @@
...
+    <input type="text"><button>OK</button>
     <script src="./main.js"></script>
```
Comment (eranshmil) [inline dist/index.html]: adasdasd

**Pull request merged**